src/queue/index.ts

```diff
@@ -182,6 +180,19 @@ function internalPlayerSystem() {
     }
   }
 }
+
+let inactiveSince = 0
+function inputsCheckTimer() {
+  return function () {
+    const { inputSystem } = getSDK()
+
+    //update player inactivty timer
+    if (inputSystem.isTriggered(InputAction.IA_ANY, PointerEventType.PET_DOWN)) {
```

Comment:
maybe we should add this to only the active players? 

What if a player that is in the queue is waiting to play and doesn't trigger any key event. When its his turn it will be kick off inmediatly because this system was running when it was on the queue.